lib/doc/doxygen/snippets/CMakeFiles/CMakeDirectoryInformation.cmake

```diff
@@ -0,0 +1,16 @@
+# CMAKE generated file: DO NOT EDIT!
```

Comment:
As the comment says, this file has been created by CMake as part of an in-source build, please remove all files in the `lib/doc/doxygen/snippets` from the repository.

Reply:
Removed.